.github/workflows/zz-tmpl-images.yaml

```diff
@@ -9,10 +9,10 @@ on:
         type: string
       platforms-test:
         type: string
-        default: linux/amd64
+        default: linux/amd64,linux/arm64
```

Comment:
can you please revert this on both lines? This means for every image we will test and build against arm64, but we don't have enough resources for it, so we keep testing against amd64.

The target publishing for this image can be controlled on images.yaml workflows file